update.sh

```diff
@@ -6,8 +6,8 @@
 # Description:
 #   An update bash script for engelsystem
 
-# Add upstream 
+# Add upstream
 git remote add upstream http://www.github.com/fossasia/engelsystem.git
```

Comment:
If he forks the project and make somes changes, he should update from his repo, not FOSSASIA repo, because who can guarantee that new change in FOSSASIA won't break his system? The code update in FOSSASIA repo is only tested against the code in FOSSASIA repo, not against code of someone else. And who guarantee that new update from FOSSASIA won't have conflict with the change he made in his repo?